dissect/etl/headers/event.py

```diff
@@ -122,7 +123,7 @@ class EventHeaderExtendedDataItem:
         "raw_data",
     ]
 
-    def __init__(self, payload):
+    def __init__(self, payload: memoryview):
```

Comment:
```suggestion
    def __init__(self, payload: bytes):
```
I don't think it's restricted to a memoryview?